### PR TITLE
fixed typo in translation + added new translation strings

### PR DESCRIPTION
**Pull request by dabura667**

### CODE REVIEW

**Comment by matiu** [comment]
LGTM!

@cmgustavo ?

**Comment by cmgustavo** [comment]
I am on it. 

There is an syntax error in file importLegacy.html. Please, check it and updated your PR (or let me send the corrections to your PR).

**Comment by dabura667** [comment]
Sorry, that was the best I could do. I don't know this syntax well enough. Could you send a PR to my addjp branch please?

File: src/js/services/profileService.js [hunk 1]
```diff
@@ -165,7 +165,7 @@ angular.module('copayApp.services')
       }
 
       var walletClient = bwcService.getClient();
-      walletClient.createWallet('Personal Wallet', 'me', 1, 1, {
+      walletClient.createWallet(gettext('Personal Wallet'), gettext('me'), 1, 1, {
```
Comment (cmgustavo) [inline src/js/services/profileService.js]: It 's not working to me :( We should check it in another PR.

Comment (dabura667) [inline src/js/services/profileService.js]: Yes, I just checked too... this doesn't work... hmmm...

Comment (dabura667) [inline src/js/services/profileService.js]: I will remove the two errors for now including this one.

Comment (cmgustavo) [inline src/js/services/profileService.js]: No problem. I send you a PR with these changes. Ignore it if you already fixed it.

**Comment by dabura667** [comment]
Removed the errors.

**Comment by dabura667** [comment]
Wait a second

I will input the () and see if it works

**Comment by dabura667** [comment]
ok nevermind didn't work. Just merge this as is

**Comment by cmgustavo** [comment]
This LGTM! Nice job!

**Pull request merged**